### ISSUE TITLE
Show progress of scanning

### ISSUE THREAD

**Issue by uok** [syncthing/syncthing#1032]
During scanning Synching currently does not indicate a progress and stays this way for an indefinite amount of time. If possible, it would be great if "Scanning (XY %)" could be displayed (XY = scanned files / total files)

**Comment by 3three** [comment]
You don't need to provide a percentage complete. Simply a count of files/folders/MB that have been scanned so far would do. This is mainly for user feedback so they know that something is happening and hasn't crashed.

**Comment by MartenBE** [comment]
I've recently downloaded and installed syncthing for the first time to try it out. I wanted to share my movies folder, which is about 200 GB. The scan took very long and it was very frustrating not having any feedback about it was doing. I didn't know this was expected behavior or not: eventually the scan took too long (hours) and I had to shut the pc off, because I had to go...

It would help a lot to have a textbox which shows which files has been scanned:

scanning testtest.mp4
test.mp4 has been scanned
test.srt has been scanned
test2/test2.avi has been scanned
...

This way you don't have to pre-scan the folder, but users can still see what it is doing and won't dismiss the software as "broken" or "buggy" because they think it's stuck or don't know what it is doing.
